src/pages/index.js

```diff
@@ -1,30 +1,57 @@
-import React from "react";
+import React, { useEffect } from "react";
 // import React, { Suspense } from "react";
-import { I18nextProvider } from "react-i18next";
-
-import Layout from "../components/layout";
+// import { Redirect, Router } from "@reach/router";
+import { navigate } from "gatsby";
+import { useTranslation } from "react-i18next";
+// import { i18next } from "../components";
 import i18next from "../components/i18n";
+// import NotFound from "../components/404";
 
 import "@trussworks/react-uswds/lib/uswds.css";
 import "@trussworks/react-uswds/lib/index.css";
 
-function Home() {
+function Home({ location }) {
```

Comment:
I'm not sure if we should be redirecting directly from index.js. I think we would probably want to do this in createNodeField in the gatsby-node.js file. Also, in our discussions about routing during our short-term projects, we talked about how using split and selecting a specific hard-coded array element isn't necessarily the way to go with routes. If we change the route structure, then the splitting/selecting/joining logic would all need to be changed too. I'm not super familiar with lodash but maybe we can use that syntax here?

Also, I think we can probably delete index.js altogether. Like I mentioned in code review last week, if slug is 'home' we can go to `/${lang}/` and if slug is not home then we can go to `/${lang}/${slug}/` (this would be in createNodeField in gatsby-node.js).  What do you think?

Reply:
From the way you create the markdown pages, we cannot go to `/${lang}/` if the slug is home because the page is loaded on the route `/${lang}/home`.

Reply:
Can we change the way it is loaded in the gatsby-node file to load on `/${lang}/` when `slug = home` and load `/${lang}/${slug}/` for any other slug

Reply:
Also, handled redirects in `createPages` in the gatsby-node.js file for routes `/en/` and `/es/`.

Reply:
About your `lodash` comment, I'm not sure if we can utilize that or `@reach/router` since they require the location to be matched with the components, but we cannot do that as the component is being used as a template.

Reply:
Changed routing in `gatsby-node.js` with how you said it. Amazing! If the slug is 'home', we go to `/${lang}/` and if the slug is not 'home', then we go to `/${lang}/${slug}/`. At first, I had read your comment wrong.

Reply:
Handled redirects in `createPages` in the gatsby-node.js file for routes `/en/home` and `/es/home`.

Reply:
Awesome!!! And okay good to know about the issues with location + templates